content/home.pml

```diff
@@ -28,17 +28,17 @@
     <ul>
       <li>
         <strong>
-	  8 Jul 2021: Second State joins the seL4 Foundation
+	  13 Jul 2021: TU Munich joins the seL4 Foundation as an Associate Member
```

Comment:
"Technical University of Munich (TUM) joins ..." would be more compliant with their style guide.